Python/Tests/Core.UI/TestExplorerTests.cs

```diff
@@ -122,6 +169,31 @@ public class TestExplorerTests {
             return workspaceFolderPath;
         }
 
+        private static void DebugTest(PythonVisualStudioApp app, TestInfo test, TestInfo[] tests) {
+            var testExplorer = app.OpenTestExplorer();
+            Assert.IsNotNull(testExplorer, "Could not open test explorer");
+
+            Console.WriteLine("Waiting for tests discovery");
+            app.WaitForOutputWindowText("Tests", $"Discovery finished: {tests.Length} tests found", 15_000);
+
+            testExplorer.GroupByProjectNamespaceClass();
+
+            var item = testExplorer.WaitForItem(test.Path);
+            Assert.IsNotNull(item, $"Coult not find {string.Join(":", test.Path)}");
+
+            var breakLineno = test.SourceLine + 1;
+            app.Dte.Debugger.Breakpoints.Add(File: test.SourceFile, Line: breakLineno);
+
+            Console.WriteLine($"Debugging test {test.Name}");
+
+            testExplorer.DebugAll();
+
+            app.WaitForMode(EnvDTE.dbgDebugMode.dbgBreakMode);
+
+            Assert.IsNotNull(app.Dte.Debugger.BreakpointLastHit);
+            Assert.AreEqual(breakLineno, app.Dte.Debugger.BreakpointLastHit.FileLine);
```

Comment:
currently only checking breakpoint was hit and line.  Couldn't find a way to check which debugger is loaded.